subsys/sd/sdmmc.c

```diff
@@ -387,28 +395,27 @@ static int sdmmc_set_current_limit(struct sd_card *card)
 	int max_current = -1;
 	uint8_t *status = card->card_buffer;
 
-	if ((card->card_speed != SD_TIMING_SDR50) &&
-		(card->card_speed != SD_TIMING_SDR104) &&
-		(card->card_speed != SD_TIMING_DDR50)) {
+	if ((card->card_speed != SD_TIMING_SDR50) && (card->card_speed != SD_TIMING_SDR104) &&
+	    (card->card_speed != SD_TIMING_DDR50)) {
 		return 0; /* Cannot set current limit */
 	} else if (card->host_props.max_current_180 >= 800 &&
-		(card->switch_caps.sd_current_limit & SD_MAX_CURRENT_800MA)) {
+		   (card->switch_caps.sd_current_limit & SD_MAX_CURRENT_800MA)) {
 		max_current = SD_SET_CURRENT_800MA;
 	} else if (card->host_props.max_current_180 >= 600 &&
-		(card->switch_caps.sd_current_limit & SD_MAX_CURRENT_600MA)) {
+		   (card->switch_caps.sd_current_limit & SD_MAX_CURRENT_600MA)) {
 		max_current = SD_SET_CURRENT_600MA;
 	} else if (card->host_props.max_current_180 >= 400 &&
-		(card->switch_caps.sd_current_limit & SD_MAX_CURRENT_400MA)) {
+		   (card->switch_caps.sd_current_limit & SD_MAX_CURRENT_400MA)) {
```

Comment:
Are you running `clang-format` on this file? If you'd like to do so you are welcome to, but please make a commit with *just* those changes. Putting both changes in one commit makes this type of otherwise simple fix difficult to review

Reply:
Sure, will do!